### PR TITLE
set up server

### PR DESCRIPTION
**Pull request by kingsley-einstein**
**What this PR does**
Sets up the initial Express server and routes to effectively leverage the Twitter API.

### CODE REVIEW

**Comment by proful** [comment]
Awesome work @kingsley-einstein

I am thinking to use AWS serverlss Lambda.

**Comment by tr0mbl3y** [comment]
@kingsley-einstein : it wud be an added advantage if you could demonstrate the API routing part through a  short video

**Comment by kingsley-einstein** [comment]
> Awesome work @kingsley-einstein
> 
> I am thinking to use AWS serverlss Lambda.
> 
> Move the project under playground/cli./server

@proful 
This isn't the CLI project. I haven't started that yet.

**Comment by kingsley-einstein** [comment]
> @kingsley-einstein : it wud be an added advantage if you could demonstrate the API routing part through a short video

@tr0mbl3y 

I would do that. I just got my Twitter's developer's account approved, so I'll run the app on my local machine and come up with a video.

**Comment by proful** [comment]
I get it. Can you come to discord and call me?

File: server/api/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const { default: Axios } = require("axios");
+
+class API {
```
Comment (johnjacobkenny) [inline server/api/index.js:3]: Curious whether to use a class with static, or just an module exports a few functions

**Review by johnjacobkenny — APPROVED** [review]
@kingsley-einstein is the bomb :partying_face:

**Comment by tr0mbl3y** [comment]
twindlelogo
by: @tr0mbley_

**Review by piratelicorne — APPROVED** [review]

**Comment by piratelicorne** [comment]
@kingsley-einstein just check the conflict.

File: server/api/index.js (hunk 1 above)
Comment (kingsley-einstein) [reply server/api/index.js:3]: I'll be glad to implement any way you feel is better.

**Comment by kingsley-einstein** [comment]
@piratelicorne  Conflicts have been resolved
@proful Are you still in for the call? I'm sorry my response came in late.

**Pull request merged**